learn/howtos-documents.md

```diff
@@ -85,6 +85,7 @@ Find the how to topics and look for sub-categories under each of the specified c
 *   [How to schedule a Java Service](/learn/how-tos/scheduling-java-service/)
 *   [Custom Query Syntax to access DB Service APIs from Java Service](/learn/app-development/services/database-services/orm-artifacts/#custom-query-syntax)
 *   [How to accomplish Pre-Post Processing for a DB Service APIs](/learn/how-tos/pre-post-processing-db-service-apis/)
+*   [How to Integrate Camunda with WaveMaker Apps](/learn/how-tos/camunda-wavemaker-app-integration/)
```

Comment:
Lets change this to How to design a workflow in Camunda BPM and integrate into a WaveMaker app

Reply:
yes done.